packages/track-component/index.js

```diff
@@ -62,10 +62,12 @@ class Track extends Component {
       <li tabindex=0 class="track-component flex items-center w-100 mb2" onkeypress=${this._handleKeyPress}>
         <div class="flex items-center flex-auto">
           ${this.renderPlaybackButton()}
-          <div ondblclick=${this._handleDoubleClick} class="metas no-underline truncate flex flex-column pl2 pr2 items-start justify-center w-100">
-            <span class="pa0 track-title truncate f5 w-100">
-              ${this.local.track.title}
-            </span>
+          <div ondblclick=${this._handleClick} class="metas no-underline truncate flex flex-column pl2 pr2 items-start justify-center w-100">
```

Comment:
Is the double-click handler still needed? Can it just be put on the same div as the single-click handler if so?

Reply:
Nice catch, I addressed this here: https://github.com/resonatecoop/stream/pull/212/commits/3b48df7538b5dfdc790ce987f8cdf95188f23a6f